test/factories/statistics_announcements.rb

```diff
@@ -12,22 +12,22 @@
     organisations { FactoryBot.build_list :organisation, 1 }
 
     association :creator, factory: :writer
-    association :current_release_date, factory: :statistics_announcement_date
+    statistics_announcement_dates { [create(:statistics_announcement_date)] }
```

Comment:
I reckon this could probably be `build` rather than `create`? 🤔 

```suggestion
    statistics_announcement_dates { build_list(:statistics_announcement_date, 1) }
```

Reply:
good shout. Have sorted